### PR TITLE
chore: auto request all reviewers

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/4092

Note: hasn't been tested yet. Seems like something we can verify post merge.

### CODE REVIEW

File: .github/auto_request_review.yml
Comment (rootulp) [inline .github/auto_request_review.yml:4]: @cmwaters LMK if you would like to be removed from this list. If that's the case, we can explicitly add you to PRs that need your attention

File: .github/CODEOWNERS
Comment (rootulp) [inline .github/CODEOWNERS:18]: I removed these b/c I think they outlived their purpose

**Review by rach-id — APPROVED** [review]
Approving but Im wondering if there was a decision to make all reviewers review all PRs?

**Comment by rootulp** [comment]
> Approving but Im wondering if there was a decision to make all reviewers review all PRs?

There wasn't a decision so this is up for debate. Currently we have a rule that each PR must have 2 approvals before being merged. Since 2 random ppl get tagged on each PR, it can take a while for those two people to review. I usually add all core/app engineers as reviewers on my PRs so I figured I'd automate it.

File: .github/auto_request_review.yml
Comment (cmwaters) [reply .github/auto_request_review.yml:4]: I'm okay with this for now. I'm not too sensitive to github notifications

**Review by cmwaters — APPROVED** [review]
I prefer everyone being tagged for reviews. Everyone that has the capacity should spend time reviewing everyone else's work. It's super important.

**Pull request merged**
